packages/plugin-parser/src/ParsingEngine.ts

```diff
@@ -84,6 +105,7 @@ export type ParsingEngineConstructor<T = {}> = {
     new (...args: ConstructorParameters<typeof ParsingEngine>): ParsingEngine;
     id: ParsingIdentifier;
     defaultParser: ParserConstructor<T>;
+    fallbacks?: ParsingIdentifier[];
```

Comment:
If we rename `fallbacks` to `extends` then this one will need to be renamed as well. And please move it alongside `id` and before `defaultParser`.